### ISSUE TITLE
about fov of eg3d 

### ISSUE THREAD

**Issue by dwliu1412** [hse1032/GSGAN#1]
Hi authors, thanks for your great work. I see you set the fov=12.96, and explain it as 'TODO EG3D assumes focal length 4.26 x image width, this makes fov ~ 12.96‘. But the results I calculated using the following formula are inconsistent：fov=2*math.atan(1/2/4.26)*(180/math.pi) = 13.388

**Comment by hse1032** [comment]
Thank you for bringing this to our attention.

It's possible that I made a mistake when calculating the FOV (maybe, this is just my mistake), 
and we have trained our model with current FOV as in this implementation.

I'm unsure how much this might have affected the model's performance or training convergence.

I'll review it again to ensure the performance.

**Comment by ramazan793** [comment]
> fov=12.96

Isn't this pov 12.96 is in radians? Because it later used in `renderer.py` which uses `math.tan`, which accepts angle measured in radians:
```
tanfovx = math.tan(viewpoint_camera.FoVx * 0.5)
tanfovy = math.tan(viewpoint_camera.FoVy * 0.5)
```
So, the correct fov should be: `fov=2math.atan(1/2/4.26)=0.2336`. 
Am I missing something?

**Comment by hse1032** [comment]
Well, it seems @ramazan793 is correct.

The current model uses (12.96 % math.pi) radian as fov, which equates to roughly 22 degrees.
Now I just launched the script for training the generator on FFHQ-256 with modified fov (0.2336 radians), and it seems to be trained well.

I will update the results when it converges to some extent.

Thanks for both @ramazan793 and @KID-1412-git!

**Comment by hse1032** [comment]
I'm sorry for not getting back sooner.

I trained the model with modified fov (0.2336 radians) with some changes in hyperparameters as below:
- Modify fov parameter
- Halve the scale of initial Gaussians (log(1/resolution) --> log(1/(resolution * 2))
- Increase the number of initial Gaussians (256 --> 384)
- Increase the coefficient of center loss(1 --> 10)

With this configuration, I trained the model on 10M images.
The model is successfully trained on FFHQ-256 dataset, achieving a similar FID (6.5~6.6).
I did not test the hyperparameters much, but it seems that increasing the number of initial Gaussians would be needed for more stable training with modified fov.

The below videos are the generated examples.

https://github.com/user-attachments/assets/8a67b616-6f43-4ea7-9066-79cd2459687d

Also, I will upload the trained model on google drive link for pretrained models, so please check it if you interests in the models with modified fov.

Thanks for valuable comments again!
